lunatrace/bsl/frontend/src/pages/project/settings/PrComments.tsx

```diff
@@ -26,39 +27,45 @@ export const SettingsPrComments: React.FC<SettingsPRCommentsProps> = ({ project
     return null;
   }
 
-  const [updateSettings, { isLoading }] = api.useUpdateSettingsMutation();
+  const [updateSettings, { isLoading }] = api.useUpsertProjectSettingsMutation();
 
   // Allows us to show the change before it syncs to the server. A bit easier than dealing with manually editing the store cache
   // to show optimistic updates
-  const [checked, setChecked] = useState<boolean>(project.settings?.pr_feedback_enabled || false);
+  const [checked, setChecked] = useState<boolean>(project.settings?.pr_feedback_disabled || false);
 
   // We have specified that updateSettings will bust the ProjectDetails cache, so we expect project to be refetched and
   // then we sync the state from its settings
   useEffect(() => {
-    setChecked(project.settings?.pr_feedback_enabled || false);
+    setChecked(project.settings?.pr_feedback_disabled || false);
   }, [project]);
 
   return (
     <>
       <Row>
         <Col md="4">
-          <h2>Pull Request Feedback</h2>
+          <h3>Pull Request Feedback</h3>
         </Col>
         <Col md>
-          <p>Whether LunaTrace will automatically submit feedback your PRs.</p>
+          <p>Disable LunaTrace from submitting feedback to your PRs.</p>
```

Comment:
This is a confusing UX now.  So PR comments are only ON when the switch is OFF? It's backwards.

"Dont make me think"  

I'm perfectly happy to change the underlying data structure to be less semantic if it enables some useful business logic, but there's no reason to have a confusing UX.